### PR TITLE
(DOCSP-11373, DOCSP-11380, DOCSP-11387) Update Copyright and Trademark

### PR DESCRIPTION
**Pull request by atsansone**
@melissamahoney-mongodb : Made [changes](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-11373/style-guide/style/copyrights.html) per our conversation with Legal.

---
[Builds without errors on `master`.](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=5f34349f2f9a5cfb5bb3117d)

### CODE REVIEW

File: source/style-guide/style/copyrights.txt [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
-==========
-Copyrights
-==========
+=========================
+Copyrights and Trademarks
+=========================
 
-.. include:: /style-guide/includes/page-needs-update.rst
+- **Don't** include copyright or trademark symbols after a trademark or
+  trade name.
```
Comment (melissamahoney-mongodb) [inline source/style-guide/style/copyrights.txt:10]: I wouldn't include this.

Comment (atsansone) [reply source/style-guide/style/copyrights.txt:10]: **question:** Why not? He did say that we don't need to do it.

Comment (melissamahoney-mongodb) [reply source/style-guide/style/copyrights.txt:10]: And this sticks more closely to his written guidance: only talk about logos.

Comment (atsansone) [reply source/style-guide/style/copyrights.txt]: Updated.

**Comment by atsansone** [comment]
Reviewed and approved by legal and Grace.

**Pull request merged**